packages/components/src/components/content-expander/content-expander.jsx

```diff
@@ -28,6 +29,12 @@ const ContentExpander = ({
         }
     }, [is_visible, onToggle, toggleVisibility]);
 
+    React.useEffect(() => {
+        if (measure && typeof measure === 'function') {
```

Comment:
Isn't `typeof measure === 'function'` enough?

Reply:
Thanks for review 
right we don't need to check it 👍